src/main/java/frc/robot/commands/ScoreAmp.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.Elevator;
+import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.ShooterPivot;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class ScoreAmp extends SequentialCommandGroup {
+  /** Creates a new ScoreAmp. */
+  public ScoreAmp(Shooter shooter, Elevator elevator, ShooterPivot shooterPivot) {
+    // Add your commands in the addCommands() call, e.g.
+    // addCommands(new FooCommand(), new BarCommand());
+    addCommands(
+        new InstantCommand(shooter::turnOnFeeder)
```

Comment:
This will push the note the wrong way.

Reply:
We will want a new speed constant for the AMP shot. It is not necessarily the same as the intake speed.